dom0/sd-svs-disp-files.sls

```diff
@@ -19,3 +19,18 @@ sd-svs-disp-install-mimetype-handler-package:
       - securedrop-workstation-svs-disp
     - require:
       - sls: fpf-apt-test-repo
+
+paxctld:
+  service.running:
+    - enable: True
+    - reload: True
+
+sd-svs-disp-install-libreoffice:
+  pkg.installed:
+    - name: libreoffice
+    - retry:
+        attempts: 3
+        interval: 60
+    - install_recommends: False
```

Comment:
Given that the `install_recommends: False` sidesteps the PaX flags problem, do we still need the retry logic? Doesn't hurt to keep it in, but it seems like we resolve the problem without resorting to retries.